### PR TITLE
PORTAL-3850 Move getLocalizationString from sdk to bridge

### PR DESCRIPTION
**Pull request by Zack0711**
- Move getLocalizationString from SDK to bridge. 
- Modify LocalizableContent, CurrentContext, and  CurrentPlayer type definition. Make them align with current data structure.	

tickets:
https://rgames.atlassian.net/browse/PORTAL-3850
https://rgames.atlassian.net/browse/PORTAL-4187

### CODE REVIEW

File: src/types/context.ts [hunk 1]
```diff
@@ -51,5 +52,5 @@ export interface CurrentContext {
   id: string | null;
   type: 'SOLO' | 'THREAD';
   size: number;
-  connectedPlayers: ContextPlayer[];
-}
+  connectedPlayers: ContextPlayer[] | PlayerRawData[];
```
Comment (toruta39) [inline src/types/context.ts:55]: `ContextPlayer` is instance with methods and `PlayerRawData` is just plain object. Can I ask to align with the old interface with only instances?

File: src/sdk/context.ts [hunk 2]
```diff
@@ -297,7 +278,7 @@ export function chooseAsync (payload: ContextChoosePayload): Promise<void> {
  * });
  * ```
  */
-export function getPlayersAsync (): Promise<ContextPlayer[]> {
+export function getPlayersAsync (): Promise<PlayerRawData[] | ContextPlayer[]> {
```
Comment (toruta39) [inline src/sdk/context.ts:281]: Ditto

File: src/types/player.ts [hunk 3]
```diff
@@ -17,5 +17,5 @@ export interface CurrentPlayer {
   name: string | null;
   id: string | null;
   photo: string | null;
-  connectedPlayers: ConnectedPlayer[];
+  connectedPlayers?: ConnectedPlayer[];
```
Comment (toruta39) [inline src/types/player.ts:20]: this property can't be null, so please keep it without the `?`

File: src/types/localizable-content.ts [hunk 4]
```diff
@@ -5,5 +5,5 @@
 export interface LocalizableContent {
   /** Text will be used if not finding matching locale */
   default: string,
-  [locale: string]: string,
+  localizations: Record<string, string>
```
Comment (toruta39) [inline src/types/localizable-content.ts:8]: thanks for fixing the wrong typing

**Review by toruta39 — CHANGES_REQUESTED** [review]
Find some unexpected breaking changes in current implementation. Please double check.

**Comment by Zack0711** [comment]
Changed after review
- Keep CurrentPlayer and CurrentContext type definition.
- Add InitializeResponsePlayer and InitializeResponseContext definition for InitializeResponse.
- Covert InitializeResponse connected player to ContextPlayer instance.

**Review by toruta39 — APPROVED** [review]
LGTM

**Pull request merged**